containers/Dockerfile.buildenv

```diff
@@ -342,6 +339,12 @@ RUN wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add - \
     && add-apt-repository 'deb http://apt.llvm.org/focal/ llvm-toolchain-focal-13 main' \
     && apt-get update -y && apt-get install -y clang-13
 
+# Install minimum required CMake version so that we test compatibility with it
+RUN wget https://github.com/Kitware/CMake/releases/download/v3.18.2/cmake-3.18.2-linux-${CMAKE_ARCH}.sh -O cmake-install.sh \
```

Comment:
Sorry, what line above causes this to be skipped on ARM?

Reply:
This isn't skipped on ARM. Currently we don't build the CI container on ARM at all. This step (installing CMake v3.18) would be one of the steps that fail (because Kitware doesn't publish a v3.18 ARM binary). I haven't actually tried getting the CI container to build on ARM.

Reply:
I added a conditional to install this only on x86 and checked that the CI container builds on ARM. We still don't build it on CI or push it to Dockerhub though, because currently GitHub doesn't have any ARM runners.